protocols/v2/binary-sv2/serde-sv2/src/primitives/byte_arrays/b016m.rs

```diff
@@ -4,7 +4,7 @@ use alloc::vec::Vec;
 use core::convert::TryFrom;
```

Comment:
There is no need to change this file. This file should remain exactly equal to the one that is the main branch.